exhibitor-core/src/main/java/com/netflix/exhibitor/core/config/azure/AzurePseudoLock.java

```diff
@@ -0,0 +1,76 @@
+package com.netflix.exhibitor.core.config.azure;
+
+import com.microsoft.azure.storage.StorageException;
+import com.microsoft.azure.storage.blob.CloudBlockBlob;
+import com.microsoft.azure.storage.blob.ListBlobItem;
+import com.netflix.exhibitor.core.azure.AzureClient;
+import com.netflix.exhibitor.core.config.PseudoLockBase;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class AzurePseudoLock extends PseudoLockBase
+{
+    private final AzureClient      client;
+    private final String        container;
+
+    /**
+     * @param client the Azure client
```

Comment:
I really have an issue with this kind of Javadoc (sorry :) )
yes, `container` is a bit of a giveaway that it's going to be a... "container" :)

but on the `lockPrefix` - it would be worth to elaborate on the point, maybe?
also, could we please call the timeout `timeoutMsec`? (but, kudos for using the `Ms` suffix: this is the one thing I find myself commenting most often!)